command/git.go

```diff
@@ -268,6 +281,17 @@ func (c *gitFetchProject) Execute(ctx context.Context,
 		return errors.Wrap(err, "problem running fetch command")
 	}
 
+	var p *patch.Patch
+	if evergreen.IsPatchRequester(conf.Task.Requester) {
+		logger.Execution().Info("Fetching patch.")
+		p, err = comm.GetTaskPatch(ctx, client.TaskData{ID: conf.Task.Id, Secret: conf.Task.Secret})
+		if err != nil {
+			err = errors.Wrap(err, "Failed to get patch")
+			logger.Execution().Error(err.Error())
```

Comment:
I worry about log-and-return situations... why not just trust the caller to log the error.

also you can just log the error without calling .Error() and it all works.

Reply:
done.